subsys/bluetooth/common/dummy.c

```diff
@@ -29,12 +29,22 @@ BUILD_ASSERT(CONFIG_BT_HCI_TX_PRIO < CONFIG_BT_RX_PRIO);
 BUILD_ASSERT(CONFIG_BT_CTLR_RX_PRIO < CONFIG_BT_HCI_TX_PRIO);
 #endif /* CONFIG_BT_CTLR */
 
-/* Immediate logging is not supported with the software-based Link Layer
- * since it introduces ISR latency due to outputting log messages with
- * interrupts disabled.
+#if (defined(CONFIG_LOG_BACKEND_RTT) &&                     \
+     (defined(CONFIG_SEGGER_RTT_MODE_BLOCK_IF_FIFO_FULL) || \
+      defined(CONFIG_LOG_BACKEND_RTT_MODE_BLOCK))) ||       \
+    defined(CONFIG_LOG_BACKEND_NET) ||                      \
+    defined(CONFIG_LOG_IMMEDIATE_CLEAN_OUTPUT)
+#define INCOMPATIBLE_IMMEDIATE_LOG_BACKEND 1
+#endif
+
+/* Immediate logging on most backend is not supported
+ * with the software-based Link Layer since it introduces ISR latency
+ * due to outputting log messages with interrupts disabled.
  */
 #if !defined(CONFIG_TEST) && !defined(CONFIG_ARCH_POSIX) && \
-	defined(CONFIG_BT_LL_SW_SPLIT)
-BUILD_ASSERT(!IS_ENABLED(CONFIG_LOG_IMMEDIATE), "Immediate logging not "
-	     "supported with the software Link Layer");
+	defined(CONFIG_BT_LL_SW_SPLIT) &&                       \
+	defined(INCOMPATIBLE_IMMEDIATE_LOG_BACKEND)
+BUILD_ASSERT(!IS_ENABLED(CONFIG_LOG_IMMEDIATE), "Immediate logging "
+												"on selected backend(s) not "
```

Comment:
This is still indented using 12 tabs. With tab-size of 8 this exceeding the 80 char limit.

Should be using 6 tabs (or possibly 1 tab + 5 spaces).

Reply:
I went for the 1 tab + 5 spaces